Umutbranch

umutgun17 opened the pull request:
I completed my API and pushed it. There is no problem on me. Can you review it, please? After that, I will merge it with master.

salih997 commented:
I reviewed your code and tests. The code works well but you need to revise your third test case.

umutgun17 commented:
Thank you for reviewing @salih997 
I fixed them and I added url files into my app folder. 
The project has been changed to home url  patterns.
I changed project name 'myapp' to 'exchangeRateAPI'

Can you check it again, please? If there is no error, I will delete conflict files and merge it.
After merging process, I will update my url and settings path on group5_api files.

salih997 commented:
It all works now. Thank you!! 👍
